standalone-metastore/metastore-common/pom.xml

```diff
@@ -333,6 +333,27 @@
       <artifactId>slf4j-simple</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+      <version>5.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.core5</groupId>
+      <artifactId>httpcore5</artifactId>
+      <version>5.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.core5</groupId>
+      <artifactId>httpcore5-h2</artifactId>
+      <version>5.3</version>
```

Comment:
The version 5.3 is used in several pom.xml files, should we define a variable in root pom.xml like `<httpcomponents.client5.version>5.3</httpcomponents.client5.version>`
and use it everywhere the version `5.3` is hardcode now?